src/init.cpp

```diff
@@ -291,6 +291,9 @@ void updateConfiguration(const pika::program_options::variables_map& vm, configu
   // These are added automatically by updateConfigurationValue.
   auto& param = getTuneParameters();
   // clang-format off
+  updateConfigurationValue(vm, param.tfactor_nthreads, "TFACTOR_NTHREADS", "tfactor-nthreads");
+  updateConfigurationValue(vm, param.tfactor_nstreams, "TFACTOR_NSTREAMS", "tfactor-nstreams");
```

Comment:
```suggestion
  updateConfigurationValue(vm, param.tfactor_num_threads, "TFACTOR_NUM_THREADS", "tfactor-num-threads");
  updateConfigurationValue(vm, param.tfactor_num_streams, "TFACTOR_NUM_STREAMS", "tfactor-num-streams");
```
? I see we've been inconsistent about this already, so could go either way :cry: We've already got `nworkers` for tune parameters, and `num_streams/handles` for config options...

Reply:
Thanks for the change @albestro. I suppose we might have to consider whether we should separately change the other `*nworkers` tune parameters to `*num_workers` as well. It's a breaking change, but perhaps worth again at this point. One can still set both variables for now to be compatible with both?